app/views/shared/_determinations_form.html.haml

```diff
@@ -50,47 +51,51 @@
             = validation_error_message(@error_presenter, :rejected_reason)
           %a#rejected_reason
-          %legend.bold-normal.form-label#checkbox-control-legend-1
+          %legend#checkbox-control-legend-1.bold-normal.form-label
             = t('.reason_for_rejection')
           = collection_check_boxes(nil, :state_reason, ClaimStateTransitionReason.reject_reasons_for(@claim), :code, :description) do |b|
             .multiple-choice
-              = b.check_box(checked: params[:state_reason].present? && params[:state_reason].include?(b.value), "aria-labelledby": "checkbox-control-legend-1 #{b.text.to_css_class}")
-              = b.label(id: b.text.to_css_class) { b.text }
+              = b.check_box(checked: params[:state_reason].present? && params[:state_reason].include?(b.value), 'aria-labelledby': "checkbox-control-legend-1 #{b.text.to_css_class}")
+              = b.label { b.text }
 
-          %fieldset.form-group.nested-fields.fieldset.spacer.js-reject-reason-text{style: 'display:none; padding-top:10px;', class: error_class?(@error_presenter, :rejected_reason_other)}
+          .form-group.panel.nested-fields.fieldset.spacer.js-reject-reason-text{ style: 'display: none; padding-top: 10px;', class: error_class?(@error_presenter, :rejected_reason_other) }
             %div
               %a#rejected_reason_other
-              = f.label :reject_reason_text, t('.reason_text')
-              .form-hint.xsmall
-                = t('.reason_hint')
-              = f.text_field :reject_reason_text, { class: 'form-control' }
+              = f.label :reject_reason_text, class: 'form-label' do
+                = t('.reason_text')
+                .form-hint.xsmall
+                  = t('.reason_hint')
+              = f.text_field :reject_reason_text, class: 'form-control'
               %div
                 = validation_error_message(@error_presenter, :rejected_reason_other)
 
       .js-cw-claim-refuse-reasons{ style: 'display:none', class: error_class?(@error_presenter, :refused_reason) }
+        %div
+          = validation_error_message(@error_presenter, :refused_reason)
+
         %fieldset.form-group.nested-fields.indent-fieldset.spacer{ 'aria-describedby': 'checkbox-control-legend-2' }
-          %div
-            = validation_error_message(@error_presenter, :refused_reason)
-          %a#refused_reason
-          %legend.bold-normal.form-label#checkbox-control-legend-2
+          %legend#checkbox-control-legend-2.bold-normal.form-label
             = t('.reason_for_refusal')
 
+          %a#refused_reason
           = collection_check_boxes(nil, :state_reason, ClaimStateTransitionReason.refuse_reasons_for(@claim), :code, :description) do |b|
             .multiple-choice
               = b.check_box(checked: params[:state_reason].present? && params[:state_reason].include?(b.value), 'aria-labelledby': "checkbox-control-legend-2 #{b.text.to_css_class}")
-              = b.label(id: b.text.to_css_class) { b.text }
```

Comment:
I don't recall why this was used but it is common and i'd be wary of removing. Feature tests may not cover it sufficiently. Have you checked feature tests cover it and/or manually tested this determination reasons feature.

Reply:
Good point, I have test and everything seems ok